Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -381,7 +381,7 @@ - (void)textInputDidBeginEditing
 
   [_eventDispatcher sendTextEventWithType:RCTTextEventTypeFocus
                                  reactTag:self.reactTag
-                                     text:[self.backedTextInputView.attributedText.string copy] // [TODO(macOS Candidate ISS#2710739)
```

Comment:
What's the reason for removing this TODO? Was our fixed pushed upstream?

Reply:
Same question for the other removed TODOs in this file

Reply:
Yes, with 15dda0ab5a491dcc83539f9ef32c9896be41074a.